### DISCUSSION TITLE
when 1.33 is planned to be released?

### DISCUSSION THREAD

**Discussion by yosefy** [nginx/unit#1364] [General]
i couldn't find any info on it

**Comment by callahad** [comment]
To make sure we're putting this in the right place, where did you look / where would you expect to find plans for the next release?

(We're aiming for week of August 19th, give or take a week)

**Comment by yosefy** [comment]
Actually, I have looked at news on the unit site, and I was looking to find
some roadmap here in conversations. I guess I have missed it somehow as
usual :)
I am asking because I am waiting for the URI rewrite fix.
Thank you for the update.

**Reply by ac000** [reply]
@yosefy You can follow the 1.33.0 release progress [here](https://github.com/nginx/unit/milestone/4).

**Comment by yosefy** [comment]
Thanks

**Comment by Tenzer** [comment]
It has been released now!
